src/elements/content-sidebar/activity-feed/activity-feed/messages.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @flow
+ * @file i18n messages
+ * @author Box
+ */
+
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+    feedInlineErrorTitle: {
+        id: 'be.activitySidebar.activityFeed.feedInlineErrorTitle',
+        defaultMessage: 'Error',
+        description: 'Error title',
+    },
+    taskDelete: {
+        id: 'be.activitySidebar.activityFeed.taskDelete',
```

Comment:
Should this be something like `be.activitySidebar.activityFeed.taskMissingError`?